CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Features
 
+- `[jest-environment-jsdom, jest-environment-node]` [**BREAKING**] Add default `exportConditions` for both `jsdom` and `node` environments ([#11924](https://github.com/facebook/jest/pull/11924))
```

Comment:
can split this into 2 entries, and mention which conditions are added

Reply:
Done :-)